### ISSUE TITLE
Odd behaviour of b alias for nested parentheses with new line after first open parentheses

### ISSUE THREAD

**Issue by Tsteinroesland** [echasnovski/mini.nvim#1318]
### Contributing guidelines

- [X] I have read [CONTRIBUTING.md](https://github.com/echasnovski/mini.nvim/blob/main/CONTRIBUTING.md)
- [X] I have read [CODE_OF_CONDUCT.md](https://github.com/echasnovski/mini.nvim/blob/main/CODE_OF_CONDUCT.md)
- [X] I have updated 'mini.nvim' to latest version

### Module(s)

mini.ai

### Description

When nesting parentheses like shown below and using <motion>ab with the cursor on the first opening parentheses, the inner set of parentheses is selected. Is this a bug or the intended behaviour?

(()
)

Motion: vab

### Neovim version

0.10.2

### Steps to reproduce

1. `nvim -nu minimal.lua`
2. position cursor on first parentheses in the following expression:
(()
)

3: vab

### Expected behavior

(()
)

### Actual behavior

()

**Comment by echasnovski** [comment]
Yes, this is an intended behavior with the default 'cover_or_next' search method. It first [searches in current line](https://github.com/echasnovski/mini.nvim/blob/0a8a1072137d916406507c941698a4bfa9dbbe7a/doc/mini-ai.txt#L453-L456) and only then tries to search in the neighborhood.
